src/docs/cookbook/design/themes.md

```diff
@@ -10,36 +10,37 @@ next:
   path: /docs/cookbook/design/fonts
 ---
 
-In order to share colors and font styles throughout our app, we can take
-advantage of themes. There are two ways to define themes: App-wide or using
-`Theme` Widgets that define the colors and font styles for a particular part of
-our application. In fact, app-wide themes are just `Theme` Widgets created at
-the root of our apps by the `MaterialApp`!
-
-After we define a Theme, we can use it within our own Widgets. In addition, the
-Material Widgets provided by Flutter will use our Theme to set the background
+To share colors and font styles throughout an app, take
+advantage of themes. There are two ways to define themes:
+App-wide or using `Theme` widgets that define the colors and
+font styles for a particular part of the application. In fact,
+app-wide themes are just `Theme` widgets created at
+the root of an apps by the `MaterialApp`.
+
+After defining a Theme, use it within your own widgets. Flutter's
+Material widgets also use your Theme to set the background
 colors and font styles for AppBars, Buttons, Checkboxes, and more.
 
 ## Creating an app theme
 
-In order to share a Theme containing colors and font styles across our entire
-app, we can provide
+To share a Theme across an entire app, provide a
 [`ThemeData`]({{site.api}}/flutter/material/ThemeData-class.html)
 to the `MaterialApp` constructor.
 
-If no `theme` is provided, Flutter creates a fallback theme under the hood.
+If no `theme` is provided,
+Flutter creates a default theme behind the scenes.
```

Comment:
I'm not sure that "behind the scenes" is any better than "under the hood". Although I do appreciate a theater analogy more than a car one...